In [6]:
import pandas as pd

BASE = '../datasets/HR-Employee-Attrition.csv'

df = pd.read_csv(BASE)

In [8]:
df.describe(exclude="number")

Attrition BusinessTravel              Department EducationField Gender  \
count       1470           1470                    1470           1470   1470   
unique         2              3                       3              6      2   
top           No  Travel_Rarely  Research & Development  Life Sciences   Male   
freq        1233           1043                     961            606    882   

                JobRole MaritalStatus Over18 OverTime  
count              1470          1470   1470     1470  
unique                9             3      1        2  
top     Sales Executive       Married      Y       No  
freq                326           673   1470     1054

Algumas conclusões que podemos tirar dos dados não numéricos do nosso DataFrame: 

 - Todas as colunas possuem dados para as 1470 linhas (linha count) isso significa que não temos registros nulos

 - Existem:
     - 2 valores únicos para a coluna "Attrition"
     - 3 valores únicos para a coluna "BusinessTravel"
     - 3 valores únicos para a coluna "Department"
     - 6 valores únicos para a coluna "EducationField"
     - 2 valores únicos para a coluna "Gender"
     - 9 valores únicos para a coluna "JobRole"
     - 3 valores únicos para a coluna "MaritalStatus"
     - 1 valor   únicos para a colua  "Over18"
     - 2 varores únicos para a coluna "OverTime"
  
 - Maioria dos funcionários estão casados (673)
 - Maioria dos funcionários são homens (882)
 - Todos os funcionários tem mais de 18 anos (o que torna esse coluna de pouca utilidade)
 - Boa parte dos funcionários trabalham  no departamento de "Research & Development" 961
 - Não é tão comum hora extra dado que segundo a coluna "OverTime" 1054 (de 1470) diz não costumar fazer.